### PR TITLE
added bal vault

### PR DESCRIPTION
**Pull request by cjinghong**

### CODE REVIEW

File: shared/src/constants/externalAddresses.json [hunk 1]
```diff
@@ -32,7 +32,8 @@
       "rPERP-TSRY-v2": "0x27b4E71841Db6E4c58a2e64d4C5F9853614fba7D",
       "rbn": "0x80Ba81056BA048c82b7b01eB8bffE342fDe1998D",
       "verbn": "0x75F024aa6ca8f7eec23465388a661209f735B0DF",
-      "perp": "0xd0A1E359811322d97991E03f863a0C30C2cF029C"
+      "perp": "0xd0A1E359811322d97991E03f863a0C30C2cF029C",
+      "bal": "0x"
```
Comment (chuddster) [inline shared/src/constants/externalAddresses.json:36]: why is this 0x?

**Review by kenchangh — APPROVED** [review]

File: shared/src/constants/v2Deployments.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "RibbonThetaVaultETHPut": "0xEC1c50724CF7a618C6cdA6CFEa5c9064AFc98E84",
     "RibbonTreasuryVaultLogic": "0xAdb17390640d00744a49f4fB006d6b8493b3A579",
     "RibbonTreasuryVaultPERP": "0x1205dCCd60B85c327479891aa375662014AdCDC2",
+    "RibbonTreasuryVaultBAL": "0x0000000000000000000000000000000000000000",
```
Comment (chuddster) [inline shared/src/constants/v2Deployments.json:16]: why is this zero address?

File: treasury/src/constants/constants.ts [hunk 3]
```diff
@@ -13,10 +13,13 @@ export const hashCode: {
 } = {
   "rPERP-TSRY":
     "0x86658217590fbd9edf0a68c0574d138a341d027a6c2d36c4c9021d0adb149df2",
+  "rBAL-TSRY":
```
Comment (chuddster) [inline treasury/src/constants/constants.ts:16]: what is this?

File: treasury/src/constants/constants.ts [hunk 4]
```diff
@@ -13,10 +13,13 @@ export const hashCode: {
...
+    "0x7c63a217c290a6c8fe3107c122460c39fce1cd883d513582bd6e1a7b3ad5e021",
 };
 
 export const minDeposit: { [vault in TreasuryVaultOptions]: BigNumber } = {
   "rPERP-TSRY": ethers.utils.parseEther("500"),
+  "rBAL-TSRY": ethers.utils.parseEther("500"),
```
Comment (chuddster) [inline treasury/src/constants/constants.ts:22]: Screen Shot 2022-07-05 at 3 13 06 PM
min deposit actually 100

Comment (chuddster) [reply treasury/src/constants/constants.ts:22]: https://etherscan.io/address/0x2a6b048eb15c7d4ddca27db4f9a454196898a0fe#readProxyContract

Comment (cjinghong) [reply treasury/src/constants/constants.ts:22]: will fix

File: shared/src/constants/v2Deployments.json (hunk 2 above)
Comment (cjinghong) [reply shared/src/constants/v2Deployments.json:16]: these are placeholder addresses for kovan

**Review by chuddster — APPROVED** [review]

**Pull request merged**